keras_hub/src/models/retinanet/feature_pyramid_test.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2024 The KerasHub Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from absl.testing import parameterized
+from keras import ops
+
+from keras_hub.src.models.retinanet.feature_pyramid import FeaturePyramid
+from keras_hub.src.tests.test_case import TestCase
+
+
+class FeaturePyramidTest(TestCase):
+    @parameterized.named_parameters(
```

Comment:
Please add a generic basic layer tests from Kerashub `run_layer_test`
https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/tests/test_case.py#L82

you can find example of how this tests is setup in other layers tests.